db/seeds.rb

```diff
@@ -5,3 +5,11 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+Category.create(name: "Category 1")
+Category.create(name: "Category 2")
+
+Product.create(name: "Product 1", price: 999, category_id: 1)
```

Comment:
w seedach lepiej jest uzywac https://apidock.com/rails/ActiveRecord/Relation/first_or_create 

^ to dobry temat na TILa :)

Reply:
Bardzo praktyczna porada! Zdecydowanie dobry temat na TILa :)